en/components/query-builder.md

```diff
@@ -149,25 +149,34 @@ The `expressionTree` is a two-way bindable property which means a corresponding
 ## Expressions Dragging
 
 With the [*Drag & Drop*](drag-drop.md) functionality, users can easily reposition condition chips to adjust their query logic dynamically.
-#### How It Works
-1. **Click and Hold:** Click and hold on chip with `right mouse` button.
-2. **Drag to Reposition:** Drag away with mouse, a drag ghost will appear. Move it to a desired position within the query, a drop ghost will appear on the designated new position.
-3. **Drop to Apply Changes:** Release the mouse button to drop the chip in its new location.
-4. **Observe Updates:** The query logic updates dynamically to reflect the new order.
-
+Chips can be dragged along the whole query tree, including in groups and subgroups. 
 ##### Notes
 - Dragging a chip does not modify its condition, only its position.
 - In the main query, changes take effect immediately after dropping a chip. In inner queries, user has to commit the inner query after repositioning the chip. 
```

Comment:
This is valid for all operations on root and inner level and since its is not drag&drop specific I believe we could remove it